Cargo.toml

```diff
@@ -2,7 +2,6 @@
 authors = ["David Pedersen <david.pdrsn@gmail.com>"]
 categories = ["asynchronous", "network-programming", "web-programming"]
 description = "Web framework that focuses on ergonomics and modularity"
-documentation = "https://docs.rs/axum/0.1.3"
```

Comment:
docs.rs selects the latest version automatically, but I don't think crate.io sets the docs URL automatically. ([example](https://crates.io/crates/ruma))

```diff
+ documentation = "https://docs.rs/axum"
```

Reply:
I think you're blocking the script from docs.rs that adds the link on crates.io :)

Reply:
Oh, sorry! You are right.